When "bee migrate" with postgres driver, Messages shown "cannot find package "github.com/go-sql-driver/mysql""

I don't use mysql driver, but this message is shown.
I can solve it by "go get", but I do not want to get unnecessary modules.
What should I do?

```
bee migrate -driver="postgres" -conn="postgres://<name>:<password>@localhost:5432/test?sslmode=disable"
______
| ___ \
| |_/ /  ___   ___
| ___ \ / _ \ / _ \
| |_/ /|  __/|  __/
\____/  \___| \___| v1.8.4
2017/06/13 16:30:10 INFO     ▶ 0001 Using 'postgres' as 'driver'
2017/06/13 16:30:10 INFO     ▶ 0002 Using 'postgres://name:password@localhost:5432/test?sslmode=disable' as 'conn'
2017/06/13 16:30:10 INFO     ▶ 0003 Running all outstanding migrations
2017/06/13 16:30:10 ERROR    ▶ 0004 Could not build migration binary: exit status 1
2017/06/13 16:30:10 ERROR    ▶ 0005 |> m.go:9:2: cannot find package "github.com/go-sql-driver/mysql" in any of:
2017/06/13 16:30:10 ERROR    ▶ 0006 |> 	/home/testuser/go/src/github.com/testuser/beego-test/vendor/github.com/go-sql-driver/mysql (vendor tree)
2017/06/13 16:30:10 ERROR    ▶ 0007 |> 	/usr/local/go/src/github.com/go-sql-driver/mysql (from $GOROOT)
2017/06/13 16:30:10 ERROR    ▶ 0008 |> 	/home/testuser/go/src/github.com/go-sql-driver/mysql (from $GOPATH)
2017/06/13 16:30:10 WARN     ▶ 0009 Could not remove temporary file: remove m: no such file or directory
```

--- a/cmd/commands/migrate/migrate.go
+++ b/cmd/commands/migrate/migrate.go
@@ -196,6 +196,17 @@ func checkForSchemaUpdateTable(db *sql.DB, driver string) {
 	}
 }
 
+func driverImportStatement(driver string) string {
+	switch driver {
+	case "mysql":
+		return "github.com/go-sql-driver/mysql"
+	case "postgres":
+		return "github.com/lib/pq"
+	default:
+		return "github.com/go-sql-driver/mysql"
+	}
+}
+
 func showMigrationsTableSQL(driver string) string {
 	switch driver {
 	case "mysql":
@@ -263,6 +274,7 @@ func writeMigrationSourceFile(dir, source, driver, connStr string, latestTime in
 		beeLogger.Log.Fatalf("Could not create file: %s", err)
 	} else {
 		content := strings.Replace(MigrationMainTPL, "{{DBDriver}}", driver, -1)
+		content = strings.Replace(content, "{{DriverRepo}}", driverImportStatement(driver), -1)
 		content = strings.Replace(content, "{{ConnStr}}", connStr, -1)
 		content = strings.Replace(content, "{{LatestTime}}", strconv.FormatInt(latestTime, 10), -1)
 		content = strings.Replace(content, "{{LatestName}}", latestName, -1)
@@ -346,8 +358,7 @@ import(
 	"github.com/astaxie/beego/orm"
 	"github.com/astaxie/beego/migration"
 
-	_ "github.com/go-sql-driver/mysql"
-	_ "github.com/lib/pq"
+	_ "{{DriverRepo}}"
 )
 
 func init(){

